client/src/components/settings/honesty.css

```
@@ -1,3 +1,11 @@
+#honesty-policy
+  :is(button[aria-disabled], button[aria-disabled]:is(:focus, :hover)) {
```

```suggestion
  :is(button[aria-disabled="true"], button[aria-disabled="true"]:is(:focus, :hover)) {
```

Either this change needs to be made or you can't use `aria-disabled="false"` on the buttons.